frontend/src/_styles/tabler.scss

```diff
@@ -6,5 +6,19088 @@
.bg-indigo{background:#4263eb}.text-indigo{color:#4263eb!important}.bg-indigo-lt{color:#4263eb!important;background:rgba(66,99,235,.1)!important}.bg-purple{background:#ae3ec9}.text-purple{color:#ae3ec9!important}.bg-purple-lt{color:#ae3ec9!important;background:rgba(174,62,201,.1)!important}.bg-pink{background:#d6336c}.text-pink{color:#d6336c!important}.bg-pink-lt{color:#d6336c!important;background:rgba(214,51,108,.1)!important}.bg-red{background:#d63939}.text-red{color:#d63939!important}.bg-red-lt{color:#d63939!important;background:rgba(214,57,57,.1)!important}.bg-orange{background:#f76707}.text-orange{color:#f76707!important}.bg-orange-lt{color:#f76707!important;background:rgba(247,103,7,.1)!important}.bg-yellow{background:#f59f00}.text-yellow{color:#f59f00!important}.bg-yellow-lt{color:#f59f00!important;background:rgba(245,159,0,.1)!important}.bg-lime{background:#74b816}.text-lime{color:#74b816!important}.bg-lime-lt{color:#74b816!important;background:rgba(116,184,22,.1)!important}.bg-green{background:#2fb344}.text-green{color:#2fb344!important}.bg-green-lt{color:#2fb344!important;background:rgba(47,179,68,.1)!important}.bg-teal{background:#0ca678}.text-teal{color:#0ca678!important}.bg-teal-lt{color:#0ca678!important;background:rgba(12,166,120,.1)!important}.bg-cyan{background:#17a2b8}.text-cyan{color:#17a2b8!important}.bg-cyan-lt{color:#17a2b8!important;background:rgba(23,162,184,.1)!important}.bg-dark{background:#232e3c}.text-dark{color:#232e3c!important}.bg-dark-lt{color:#232e3c!important;background:rgba(35,46,60,.1)!important}.bg-muted{background:#656d77}.text-muted{color:#656d77!important}.bg-muted-lt{color:#656d77!important;background:rgba(101,109,119,.1)!important}.bg-white{background:#fff}.text-white{color:#fff!important}.bg-white-lt{color:#fff!important;background:rgba(255,255,255,.1)!important}.scrollable{overflow-x:hidden;overflow-y:auto;-webkit-overflow-scrolling:touch}.scrollable.hover{overflow-y:hidden}.scrollable.hover>*{margin-top:-1px}.scrollable.hover:active,.scrollable.hover:focus,.scrollable.hover:hover{overflow:visible;overflow-y:auto}.touch .scrollable{overflow-y:auto!important}.scroll-x,.scroll-y{overflow:hidden;-webkit-overflow-scrolling:touch}.scroll-y{overflow-y:auto}.scroll-x{overflow-x:auto}.no-scroll{overflow:hidden}.w-0{width:0!important}.h-0{height:0!important}.w-1{width:.25rem!important}.h-1{height:.25rem!important}.w-2{width:.5rem!important}.h-2{height:.5rem!important}.w-3{width:1rem!important}.h-3{height:1rem!important}.w-4{width:2rem!important}.h-4{height:2rem!important}.w-5{width:4rem!important}.h-5{height:4rem!important}.w-auto{width:auto!important}.h-auto{height:auto!important}.w-px{width:1px!important}.h-px{height:1px!important}.w-full{width:100%!important}.h-full{height:100%!important}.opacity-0{opacity:0!important}.opacity-5{opacity:.05!important}.opacity-10{opacity:.1!important}.opacity-15{opacity:.15!important}.opacity-20{opacity:.2!important}.opacity-25{opacity:.25!important}.opacity-30{opacity:.3!impo
```

Comment:
Can we revert this one as well?

Reply:
Error is being thrown from `autoprefixer` without this change